test/matrix.lean

```diff
@@ -3,11 +3,11 @@ manually ported from
 https://github.com/leanprover-community/mathlib/blob/4f4a1c875d0baa92ab5d92f3fb1bb258ad9f3e5b/test/matrix.lean
 -/
 import Mathlib.Data.Matrix.Notation
--- import linear_algebra.Matrix.determinant
 import Mathlib.GroupTheory.Perm.Fin
--- import Mathlib.Tactic.NormSwap
 import Qq
 import Std.Tactic.GuardExpr
+-- import Mathlib.LinearAlgebra.Matrix.Determinant
+-- import Mathlib.Tactic.NormSwap
```

Comment:
And here?

Reply:
I checked and `Determinant` can now be imported and the test work, so I restored it.

I erased `NormSwap`.